apps/sports-helsinki/src/domain/search/venueSearch/VenueSearch.tsx

```diff
@@ -137,6 +138,7 @@ export const SimpleVenueSearchForm: React.FC<SearchComponentType> = ({
   searchRoute = SEARCH_ROUTES.SEARCH,
 }) => {
   const { t } = useTranslation('search');
```

Comment:
Just a side note, that it would be good to use the more specific translation hook here as well to get advantage of typings, autocomplete, references, etc